public/index.html

```diff
@@ -23,20 +23,19 @@ <h1>Text-to-Speech</h1>
           <section class="tts-section">
             <label for="models">Choose a model:</label>
             <select id="models" name="models">
-              <option value="aura-asteria-en">
-                Asteria (Female US English)
+              <option value="aura-2-thalia-en">
+                Thalia (Female US English)
               </option>
-              <option value="aura-luna-en">Luna (Female US English)</option>
-              <option value="aura-stella-en">Stella (Female US English)</option>
-              <option value="aura-athena-en">Athena (Female UK English)</option>
-              <option value="aura-hera-en">Hera (Female US English)</option>
-              <option value="aura-orion-en">Orion (Male US English)</option>
-              <option value="aura-arcas-en">Arcas (Male US English)</option>
-              <option value="aura-perseus-en">Perseus (Male US English)</option>
-              <option value="aura-angus-en">Angus (Male Irish English)</option>
-              <option value="aura-orpheus-en">Orpheus (Male US English)</option>
-              <option value="aura-helios-en">Helios (Male UK English)</option>
-              <option value="aura-zeus-en">Zeus (Male US English)</option>
+              <option value="aura-2-theia-en">Theia (Female Australian English)</option>
+              <option value="aura-2-andromeda-en">Andromeda (Female US English)</option>
+              <option value="aura-2-amalthea-en">Amalthea (Female Philippino English)</option>
+              <option value="aura-2-delia-en">Delia (Female US English)</option>
+              <option value="aura-2-helena-en">Helena (Female US English)</option>
+              <option value="aura-2-draco-en">Draco (Male British English)</option>
+              <option value="aura-2-apollo-en">Apollo (Male US English)</option>
+              <option value="aura-2-aries-en">Aries (Male US English)</option>
+              <option value="aura-2-hyperion-en">Hyperion (Male Australian English)</option>
+              <option value="aura-2-arcas-en">Arcas (Male US English)</option>
```

Comment:
This is not a comprehensive list of the available models. Do we want to include *all* of the aura 2 voices?